### PR TITLE
Revamp

### PR DESCRIPTION
**Pull request by tulipdexter**
This PR does (or tries to do) the following:

- Update Node to latest version (I chose not to support v14 because of the fundamental difference between NPM v6 and v7 and the effect that has on the package-lock.
- node-sass -> dart
- Simplify build and leverage rollup more
- Remove linting: linting is fun but it can really interrupt the flow of a developer, and conforming to our linting style for a code test feels a step too far
- Change example JS module to console log instead of formatting a date
- Restructure the scss

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {myComponent} from "./components/my-component";
+
+const start = () => {
+	myComponent();
```
Comment (alexkilgour) [inline js/main.js]: Think we need to make it clear in a comment that this is just an example, so people don't get confused about the purpose of this

**Review by alexkilgour — APPROVED** [review]
lovely

File: app.js [hunk 2]
```diff
@@ -1,48 +1,48 @@
...
-app.use(function (err, req, res, next) {
-  // set locals, only providing error in development
-  res.locals.message = err.message;
-  res.locals.error = req.app.get('env') === 'development' ? err : {};
-
-  // render the error page
-  res.status(err.status || 500);
-  res.render('error');
+app.use((err, req, res, next) => {
+	console.log(err.status);
```
Comment (benjclark) [inline app.js]: did you intend to leave this in?

File: package.json [hunk 3]
```diff
@@ -2,41 +2,39 @@
...
-    "eslint-config-google": "^0.14.0",
-    "fs-extra": "^9.0.1",
-    "jest-cli": "^26.4.2",
+    "@babel/core": "^7.17.5",
+    "@babel/preset-env": "^7.16.11",
+    "@rollup/plugin-babel": "^5.3.1",
+    "@rollup/plugin-node-resolve": "^13.1.3",
+    "babel-jest": "^26.6.3",
+    "imagemin-cli": "^7.0.0",
+	"jest": "^27.4.7",
```
Comment (benjclark) [inline package.json]: indent

File: app.js (hunk 2 above)
Comment (tulipdexter) [reply app.js]: oops

**Pull request merged**